pkg/operator/certrotationcontroller/certrotationcontroller.go

```diff
@@ -131,7 +132,7 @@ func NewCertRotationController(
 			Validity:          1 * 4 * time.Hour,
 			RefreshPercentage: 0.5,
 			ClientRotation: &certrotation.ClientRotation{
-				UserInfo: &user.DefaultInfo{Name: "system:kube-scheduler"},
+				UserInfo: &user.DefaultInfo{Name: "system:kube-scheduler", Groups: []string{user.KubeScheduler}, UID: uuid.NewRandom().String()},
```

Comment:
we don't actually need to set these.  Do you need a group?

Reply:
TBH, no if we don't have a group(in kube-scheduler's case, we don't have) but I was trying to populate every field possible to find out if this was causing the issue in openshift/cluster-kube-scheduler-operator#62.